nix/goInternal.nix

```diff
@@ -17,8 +17,8 @@ pkgs.buildGoModule rec {
   doCheck = false;
 
   meta = {
-    description = "The testscript command runs testscript scripts in a fresh temporary work directory tree";
-    homepage = "https://github.com/rogpeppe/go-internal/cmd/testscript";
+    description = "The go-internal package includes various packages used by the Go team, such as testscript, txtar, and more.";
+    homepage = "https://github.com/rogpeppe/go-internal";
```

Comment:
this package includes many tools (e.g. txtar), not just testscript, so I renamed it goInternal

Reply:
Thank you for updating.  That was from an older version that I had where I was just packaging testscript from here and getting the forked txtar-c (the one from bitfield... https://github.com/bitfield/txtar-c).  

I am not sure about the next comment because I can't find a standard, but from what I can tell, it seems like most nix packages use the project name for the package name.  So, for example `go-internal` not `goInternal`.  If we do go with that setup, we would probably want to update the name of this file and the assignment on shell.nix:17.  Personally, I have no opinion on this.

Reply:
I am still a little bit uncertain what the expected case is for certain things in nix projects. They advise using [camel case for variable names](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax). This nix derivation is imported into shell.nix where it is assigned to a variable. However, it is generally expected that [package names](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming) use kebab case. Here, we are using `go-internal` as a package, so I updated the use of goInternal to go-internal. 

Additionally, it seems to be encouraged to use kebab-case for file naming as well, so I updated the filename to use kebab case.